lang_php/analyze/ast_php.ml

```diff
@@ -233,6 +233,10 @@ and special =
   | This
   (* take many different forms in PHP, eval(), call_user_func, ${}, etc. *)
   | Eval
+  (* often transformed in Var "$self" in the analysis *)
```

Comment:
Do you understand this comment or are you just copying what is being said about `This`?